debezium-server-core/src/main/java/io/debezium/server/DebeziumMetrics.java

```diff
@@ -58,21 +63,39 @@ private static ObjectName getDebeziumMbean(String context) {
     public ObjectName getSnapshotMetricsObjectName() {
 
         if (snapshotMetricsObjectName == null) {
-            snapshotMetricsObjectName = getDebeziumMbean("snapshot");
+            snapshotMetricsObjectName = getDebeziumMbean("snapshot", false);
         }
 
         return snapshotMetricsObjectName;
     }
 
+    public ObjectName getSnapshotPartitionMetricsObjectName() {
+
+        if (snapshotPartitionMetricsObjectName == null) {
+            snapshotPartitionMetricsObjectName = getDebeziumMbean("snapshot", true);
+        }
+
+        return snapshotPartitionMetricsObjectName;
+    }
+
     public ObjectName getStreamingMetricsObjectName() {
 
         if (streamingMetricsObjectName == null) {
-            streamingMetricsObjectName = getDebeziumMbean("streaming");
+            streamingMetricsObjectName = getDebeziumMbean("streaming", false);
```

Comment:
Isn't `QueueTotalCapacity` in `SqlServerStreamingTaskMetrics`, which is partitioned, so shouldn't here be `true`?

Reply:
ah, right, sorry